pkg/kubectl/cmd/scale.go

```diff
@@ -110,13 +110,24 @@ func RunScale(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []stri
 	}
 
 	resourceVersion := cmdutil.GetFlagString(cmd, "resource-version")
+	if len(resourceVersion) != 0 && len(infos) > 1 {
+		return fmt.Errorf("cannot use --resource-version with multiple controllers")
+	}
 	currentSize := cmdutil.GetFlagInt(cmd, "current-replicas")
+	if currentSize != -1 && len(infos) > 1 {
+		return fmt.Errorf("cannot use --current-replicas with multiple controllers")
+	}
 	precondition := &kubectl.ScalePrecondition{currentSize, resourceVersion}
 	retry := kubectl.NewRetryParams(kubectl.Interval, kubectl.Timeout)
-	waitForReplicas := kubectl.NewRetryParams(kubectl.Interval, kubectl.Timeout)
-	if err := scaler.Scale(info.Namespace, info.Name, uint(count), precondition, retry, waitForReplicas); err != nil {
-		return err
+
+	errs := []error{}
+	for _, info := range infos {
+		if err := scaler.Scale(info.Namespace, info.Name, uint(count), precondition, retry, nil); err != nil {
+			errs = append(errs, err)
+			continue
+		}
+		fmt.Fprintf(out, "scaled %s/%s\n", info.Namespace, info.Name)
```

Comment:
We're in the process of making the default output of all mutating commands more verbose, so we need to decide on a convention to use in create, replace, patch, etc. 

I slightly prefer noun then verb: "%s/%s scaled".

cc @JanetKuo